File: amqp/spring-cloud-stream-rabbit/src/main/java/br/com/leonardoferreira/poc/Application.java
package br.com.leonardoferreira.poc;

import br.com.leonardoferreira.poc.channel.SimpleChannel;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.stream.annotation.EnableBinding;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@SpringBootApplication
@EnableBinding(SimpleChannel.class)
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: kafka/spring-cloud-stream-integrated-spring-kafka/spring-cloud-stream/src/main/kotlin/br/com/leonardoferreira/Application.kt
package br.com.leonardoferreira

import br.com.leonardoferreira.binding.CreatedUserBinding
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.stream.annotation.EnableBinding

@SpringBootApplication
@EnableBinding(CreatedUserBinding::class)
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
